packages/desktop/components/popup/popups/ReceiveAddressPopup.svelte

```diff
@@ -42,7 +42,7 @@
 
         try {
             await generateAndStoreEvmAddressForAccounts($activeProfile.type, coinType, account)
-            pollL2BalanceForAccount($activeProfileId as string, account)
```

Comment:
This should either be set on the isc chain when constructing or passing it in as a parameter. Otherwise login and logout may cause state to bleed over, because the profileId has been updated.